clio-client/src/main/scala/org/broadinstitute/clio/client/dispatch/CommandDispatch.scala

```diff
@@ -19,12 +19,9 @@ class CommandDispatch(val webClient: ClioWebClient, val ioUtil: IoUtil)
     bearerToken: OAuth2BearerToken
   ): Future[HttpResponse] = {
     (command match {
-      case addCommand: AddCommand[_]   => new AddExecutor(addCommand)
-      case moveCommand: MoveCommand[_] => new MoveExecutor(moveCommand)
-      case deleteWgsUbam: DeleteWgsUbam =>
-        new DeleteExecutorWgsUbam(deleteWgsUbam)
-      case deleteGvcf: DeleteGvcf =>
-        new DeleteExecutorGvcf(deleteGvcf)
+      case addCommand: AddCommand[_]       => new AddExecutor(addCommand)
+      case moveCommand: MoveCommand[_]     => new MoveExecutor(moveCommand)
+      case deleteCommand: DeleteCommand[_] => new DeleteExecutor(deleteCommand)
       case other =>
         new RetrieveAndPrintExecutor(other)
```

Comment:
Could we somehow get the `match .. case` inside `RetrieveAndPrintExecutor` into this one? It seems odd to have two different matches over the same object.

Reply:
We could, but I think the end result would be more convoluted than what we've got now.